apache/incubator-zeppelin
# Remove files under /notebook/

### What is this PR for?

We ignore files under `/notebook/` but there are some existing files under that directory.

I execute the command to remove all existing files under `/notebook/` so the ignoring function would work properly.

```sh
$ git ls-files -i --exclude-from=.gitignore | xargs git rm --cached 
```

### What type of PR is it?
Improvement

### Todos
None

### What is the Jira issue?
N/A

### How should this be tested?

Check the creating or modifying files under `/notebook/` will not change the output of `git status`.

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? *no*
* Is there breaking changes for older versions? *no*
* Does this needs documentation? *no*
